app/src/main/java/com/example/coopertest/TestActivity.kt

```diff
@@ -155,6 +160,9 @@ class TestActivity : AppCompatActivity(), OnMapReadyCallback {
             override fun onFinish() {
                 mFusedLocationClient.removeLocationUpdates(mLocationCallback)
                 timerView.text = "Result:"
+                val objectResult= Results(currentDistanceMeters, objectContext.getContext() )
+                val yourLevel= objectResult.getLevel()
+                resultTextView.text=yourLevel
```

Comment:
As we thought earlier, it would be nice to also have the neighbouring levels visible to know how close you are to progress
And maybe some button leading to more detailed description. But we could add that in 'Info' activity which we need to create anyway